go/vt/vtgate/safe_session.go

```diff
@@ -22,6 +22,10 @@ import (
 	"sync"
 	"time"
 
+	"vitess.io/vitess/go/vt/sqlparser"
+
+	"vitess.io/vitess/go/vt/vtgate/engine"
+
 	"google.golang.org/protobuf/proto"
 
 	"vitess.io/vitess/go/vt/vterrors"
```

Comment:
imports for vitess should be all together

Reply:
I don't look at the imports formatting. IMHO - our formatting rules are to follow the linter. if it doesn't complain, we should fix the linter and not the code changes

It does not complain about these lines